CMakeLists.txt

```diff
@@ -127,68 +153,57 @@ if (CUDA_SUPPORT)
     # sm_75 -> Turing support
     if (DFLOAT)
         set(CMAKE_CUDA_ARCHITECTURES 52 75)
-#        set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} --gpu-architecture=compute_52 --gpu-code=sm_52,sm_75")
+        # set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} --gpu-architecture=compute_52 --gpu-code=sm_52,sm_75")
     else()
         # To use doubles we need compute capability 6.0 for atomic operations
         set(CMAKE_CUDA_ARCHITECTURES 60 75)
-#        set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} --gpu-architecture=compute_60 --gpu-code=sm_60,sm_75")
+        # set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} --gpu-architecture=compute_60 --gpu-code=sm_60,sm_75")
     endif()
 endif()
 
-
 # Memory allocators
 # ----------------------------------------------
-
-#Report on custom user header options.  This is intended to allow
-#users to provide definitions of their own memory functions
+# Report on custom user header options. This is intended to allow
+# users to provide definitions of their own memory functions
 # The header should define the functions as follows
 #
 # define c_malloc mymalloc
 # define c_calloc mycalloc
 # define c_realloc myrealloc
 # define c_free myfree
 
-if(OSQP_CUSTOM_MEMORY)
-	message(STATUS "User custom memory management header: ${OSQP_CUSTOM_MEMORY}")
+if (OSQP_CUSTOM_MEMORY)
+    message(STATUS "User custom memory management header: ${OSQP_CUSTOM_MEMORY}")
 endif()
 
-
-
 # Linear solvers dependencies
 # ---------------------------------------------
-option (ENABLE_MKL_PARDISO "Enable MKL Pardiso solver" ON)
 if (DFLOAT)
     message(STATUS "Disabling MKL Pardiso Solver with floats")
-    set(ENABLE_MKL_PARDISO OFF)
+    set(ALGEBRA_MKL OFF)
 elseif (DEFINED EMBEDDED)
     message(STATUS "Disabling MKL Pardiso Solver for embedded")
-    set(ENABLE_MKL_PARDISO OFF)
-elseif (CUDA_SUPPORT)
-    message(STATUS "Disabling MKL Pardiso Solver for CUDA")
-    set(ENABLE_MKL_PARDISO OFF)
+    set(ALGEBRA_MKL OFF)
 endif()
-message(STATUS "MKL Pardiso: ${ENABLE_MKL_PARDISO}")
-
+message(STATUS "ALGEBRA_MKL = ${ALGEBRA_MKL}")
```

Comment:
Why is this message printed here?

Reply:
I think the intention here was to indicate to the user that ALGEBRA_MKL may have been modified from what they specified on cmake invocation. This is removed now, and `ALGEBRA=..` is displayed once all decisions on what `ALGEBRA` to use have been made by our `CMakeLists.txt`